package.json

```diff
@@ -12,96 +12,99 @@
   },
   "bugs": "https://github.com/asyncapi/cli/issues",
   "dependencies": {
-    "@asyncapi/avro-schema-parser": "^3.0.23",
+    "@asyncapi/avro-schema-parser": "^3.0.24",
     "@asyncapi/bundler": "^0.6.4",
     "@asyncapi/converter": "^1.6.2",
     "@asyncapi/diff": "^0.5.0",
     "@asyncapi/generator": "^3.0.1",
-    "@asyncapi/modelina-cli": "^5.3.5",
+    "@asyncapi/generator-helpers": "^1.0.1",
+    "@asyncapi/modelina-cli": "^5.10.1",
     "@asyncapi/openapi-schema-parser": "^3.0.24",
     "@asyncapi/optimizer": "^1.0.4",
-    "@asyncapi/parser": "^3.3.0",
+    "@asyncapi/parser": "^3.4.0",
     "@asyncapi/problem": "^1.0.0",
     "@asyncapi/protobuf-schema-parser": "^3.6.0",
     "@asyncapi/raml-dt-schema-parser": "^4.0.24",
     "@asyncapi/studio": "^1.1.0",
-    "@clack/prompts": "^0.7.0",
-    "@oclif/core": "^4.2.9",
-    "@oclif/plugin-autocomplete": "^3.2.26",
+    "@clack/prompts": "^0.11.0",
+    "@oclif/core": "^4.8.0",
```

Comment:
Does this have any breaking changes?

I know it shouldn't have as it is minor version upgrade but still 🤷🏻